tests/phpunit/tests/image/editorImagick.php

```diff
@@ -783,17 +785,17 @@ public function test_resizes_are_small_for_16bit_images( $file ) {
 
 		$imagick_image_editor->resize( $size['width'] * .5, $size['height'] * .5 );
 
-		$saved = $imagick_image_editor->save( $temp_file );
+		$imagick_image_editor->save( $temp_file );
 
 		$new_filesize = filesize( $temp_file );
 
-		unlink( $temp_file );
```

Comment:
the existing tests in this file use `unlink` [ref](https://github.com/WordPress/wordpress-develop/blob/63541f8bf7fab9c66874563b7553d7312324dd20/tests/phpunit/tests/image/editorImagick.php#L583), why are you choosing to change this to `wp_delete_file`?

Reply:
reviewing unit tests, nearly all tests use `unlink`, let's stick to that convention